### PR TITLE
Make brand title dynamic

### PR DESCRIPTION
**Pull request by PhungNg**
## Description

Made brand titles switch between Swedbank Pay and Payex.
Also made some documentation examples dynamic too.

## Motivation and Context

Can not show Swedbank Pay brand when user is on Payex Design guide

## How Has This Been Tested?
Launched Payex and searched for "Swedbank Pay" on each page.
Updated snapshots. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/App/ComponentsDocumentation/components/Components/index.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const MenuOverview = () => (
 const Components = () => (
     <DocContainer>
         <p className="lead">
-        This is the Swedbank Pay component library. Here we’ve listed all the components in the Swedbank Pay design system. Feel free to look around!
+        This is our component library where we have listed all the components in our design system. Feel free to look around!
```
Comment (VikingTristan) [inline src/App/ComponentsDocumentation/components/Components/index.js:25]: nice :)

File: src/App/GettingStarted/getting-started/ForDesigners/index.js [hunk 2]
```diff
@@ -31,14 +32,14 @@ const NeedToHave = () => (
...
+        <p>When you have the right tools to get started there are some things that define the {brandTitle} Design that are good to be familiar with. We try our best to always design with our brand and identity in the back of our heads. </p>
 
         <h3>Brand and identity</h3>
         <p>Open, simple and caring. These are our values that we always strive to come back to when we design both internal and external experiences. All information about our core values and things that make us who we are can be found in <Link to="/identity/identity">Identity</Link>. </p>
-        <a href="https://brand-manager.swedbank.com/start/guidelines/swedbank-pay.html" className="icon-link" target="_blank" rel="noopener noreferrer">
+        {brandTitle === "Swedbank Pay" && <a href="https://brand-manager.swedbank.com/start/guidelines/swedbank-pay.html" className="icon-link" target="_blank" rel="noopener noreferrer">
             <i className="material-icons" aria-hidden="true">launch</i>
             <span className="ml-2">Read more about the Swedbank Pay Brand (Swedbank Brand Manager) </span>
-        </a>
+        </a>}
```
Comment (VikingTristan) [inline src/App/GettingStarted/getting-started/ForDesigners/index.js:42]: 👍

File: src/App/GettingStarted/getting-started/Introduction/index.js [hunk 3]
```diff
@@ -52,24 +53,27 @@ const UsingDG = () => (
...
+                <tbody>
+                    {designGuideUsers.map(product => (
+                        <tr key={product.name}>
+                            <td>{product.name}</td>
+                            <td><i className="material-icons mr-3">{product.icon}</i>{product.text}</td>
+                        </tr>
+                    ))}
+                </tbody>
+            </table>
+        </>}
```
Comment (VikingTristan) [inline src/App/GettingStarted/getting-started/Introduction/index.js:76]: 👍

File: src/App/Identity/identity/Accessibility/index.js [hunk 4]
```diff
@@ -4,10 +4,13 @@ import * as externalLinks from "./constants";
 import { DocContainer } from "@docutils";
 
 const basename = process.env.basename;
+const brandTitle = process.env.brandTitle;
+
+const otherLinks = brandTitle === "Swedbank Pay" ? externalLinks.othersAboutAccessibility : externalLinks.othersAboutAccessibility.slice(1);
```
Comment (VikingTristan) [inline src/App/Identity/identity/Accessibility/index.js:9]: I am skeptical about this solution but I guess it works as a quick fix for now ;)

File: src/App/Identity/identity/Copywriting/index.js [hunk 5]
```diff
@@ -220,13 +222,13 @@ const GlossaryLinks = () => (
...
-            At Swedbank Pay we care about how we’re interpreted by our customers and merchants.
-            In this section you will find an overview of the different kinds of written content relevant for Swedbank Pay,
+            At {brandTitle} we care about how we’re interpreted by our customers and merchants.
+            In this section you will find an overview of the different kinds of written content relevant for {brandTitle},
             tonality principles, a glossary, and links to further information.
         </p>
         <Terminology />
         <Tonality />
-        <GlossaryLinks />
+        {brandTitle === "Swedbank Pay" && <GlossaryLinks />}
```
Comment (VikingTristan) [inline src/App/Identity/identity/Copywriting/index.js:231]: 👍

File: src/App/Identity/identity/Identity/index.js [hunk 6]
```diff
@@ -12,12 +14,12 @@ const OurBrand = () => (
             <p>We are warm and welcoming, we guide, we involve users, we ensure access for all, we are consistent and we design for trust.</p>
         </blockquote>
 
-        <h3>Swedbank Pay design principles</h3>
+        <h3>{brandTitle} design principles</h3>
         <p className="m-0">We let our design principles reflect our brand values. When designing, we try to be warm and welcoming in our tonality. By being consistent with our interactions, involving our users, and ensuring access for all, we strive to create trust and guide our users to complete their tasks successfully.</p>
-        <a href="https://brand-manager.swedbank.com/start/guidelines/swedbank-pay.html" target="_blank" rel="noopener noreferrer"className="icon-link">
+        {brandTitle === "Swedbank pay" && <a href="https://brand-manager.swedbank.com/start/guidelines/swedbank-pay.html" target="_blank" rel="noopener noreferrer"className="icon-link">
```
Comment (VikingTristan) [inline src/App/Identity/identity/Identity/index.js:19]: 👍

**Review by VikingTristan — APPROVED** [review]
Slick!

File: src/App/Identity/identity/Accessibility/index.js (hunk 4 above)
Comment (PhungNg) [reply src/App/Identity/identity/Accessibility/index.js:9]: I know this is not scaleable at all. Will be fixed once we get some real content :)

**Pull request merged**